### ISSUE TITLE
Validate JWT token with JWKS [Oauth2]

### ISSUE THREAD

**Issue by umerasir** [wso2/product-microgateway#1198]
There is no option to provide JWKS endpoint url [https://YOUR_DOMAIN/.well-known/jwks.json] as part of micro-gw-conf to validate JWT token 

Currently on certificate from trust-store can be used to validate JWT token

```
[[jwtTokenConfig]]
  issuer = "https://localhost:9443/oauth2/token"
  audience = "http://org.wso2.apimgt/gateway"
  certificateAlias = "wso2apim310"
  validateSubscription = false
```

### How will you implement it
Give option to specify JWKS endpoint url as alternative to certificate

**Comment by praminda** [comment]
@umerasir Thanks for the suggestion. Please try to keep the issue template when creating issues. This helps us maintaining a clean/easy to manage issue list

**Comment by menakaj** [comment]
This is fixed in MGW 3.2.0.

https://github.com/wso2/product-microgateway/pull/1307

**Comment by praminda** [comment]
Duplicate of #688
